CHANGELOG.md

```diff
@@ -1,5 +1,30 @@
 # Change Log
 
+## [0.0.28] - 2023-02-13
+
+**BREAKING** changes to `IMAGE` variable
```

Comment:
Thank you for this FYI and a gentle ping to💡 @qpetraroia and @sabbour as fyi - we should scope the impact of this breaking change please before release, in case there is scenario which will be broken.

Thinking outloud - since we have moved `az-cli` and `vscode` to `1-1` consumption model hope this iwll not be impacted there.

cc: @davidgamero

Reply:
We should work on retiring the `az-cli` integration in parallel. There isn't much added value over using the `draft` binary directly and we already have an installation script and Homebrew support.

Reply:
sounds good we will make sure to verify all consumers implement the changes prior to bumping their consuming version